config/junction.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'route' => [
+        'middleware' => ['api'],
+        'prefix' => '',
+    ],
```

Comment:
Might be useful to add some comments/docks like usual is in a config to give explanation about what it does.